File: react4s/src/main/scala/samples/CommentListExample/commentList/Comment.scala
package samples.CommentListExample.commentList

import com.github.ahnfelt.react4s._

case class Comment(author: P[String], comment: P[String]) extends Component[NoEmit] {

  override def render(get: Get): ElementOrComponent = {
    E.div(
      E.p(Text("Comment Author - " + get(author))),
      E.p(Text("Comment -" + get(comment)))
    )
  }
}

File: react-redux/src/commentbox/Comment.js
import React, {Component} from 'react';

class Comment extends Component {
    render() {
        const {
            author,
            comment
        } = this.props;

        return (
            <div className="comment">
                <p>Comment Author - {author}</p>
                <p>Comment - {comment}</p>
            </div>
        );
    }
}

export default Comment;